packages/command-palette/src/commands.tsx

```diff
@@ -1383,7 +1385,7 @@ export function useCommands() {
 					_x( 'create user', 'Keyword for the Add new user command', __i18n_text_domain__ ),
 					_x( 'invite user', 'Keyword for the Add new user command', __i18n_text_domain__ ),
 					'wp user create', // WP-CLI command
-				].join( ' ' ),
```

Comment:
Should we include `wp user*` here too? Right now if I search for `wp user d` I only get the `manage users` command listed. I'm not sure if we just want to show that command or any `wp user` related command so feel free to ignore this comment.

Reply:
I don't feel strongly about this so kept inline with what we have already. 

If you search for `wp user d` (presumably you're aiming for `wp user delete`) then the 'Add new user command' doesn't seem like a good match and only listing 'Manage users" makes sense.

Listing two commands rather than one isn't too onerous for the user, but it perhaps feels slightly less fine-tuned.